web/src/components/cartSummary.js

```diff
@@ -9,12 +17,39 @@ class CartSummary extends Component {
         };
     }
 
+    itemCard = () => {
+        const listItem = [];
+        for(let i = 0; i < this.props.selected.length; i++){
+            if(this.props.returnStatusByIdx(i) !== 0){
+                listItem.push(
+                    <ListGroup.Item as="li">
+                        {this.props.returnItemNameByIdx(i)}{" "}
```

Comment:
This is not going to work when getting selected info from the server. `itemName` values in the `ShoppingBasket` are not updated when the client gets a query from the server.